gr-fec/lib/ldpc_decoder.cc

```diff
@@ -24,21 +24,20 @@
 namespace gr {
 namespace fec {
 
-generic_decoder::sptr
-ldpc_decoder::make(std::string alist_file, float sigma, int max_iterations)
+generic_decoder::sptr ldpc_decoder::make(std::string alist_file, int max_iterations)
 {
-    return generic_decoder::sptr(new ldpc_decoder(alist_file, sigma, max_iterations));
+    return generic_decoder::sptr(new ldpc_decoder(alist_file, max_iterations));
 }
 
-ldpc_decoder::ldpc_decoder(std::string alist_file, float sigma, int max_iterations)
+ldpc_decoder::ldpc_decoder(std::string alist_file, int max_iterations)
     : generic_decoder("ldpc_decoder")
 {
     if (!std::filesystem::exists(alist_file))
         throw std::runtime_error("Bad AList file name!");
 
     d_list.read(alist_file.c_str());
     d_code.set_alist(d_list);
-    d_spa.set_alist_sigma(d_list, sigma);
+    d_spa.set_alist_sigma(d_list, 1.0);
```

Comment:
yes, I was trying to do that, yet there's some API domino effect I'm not understanding because removing that function caused a segfault. I'll investigate more now